hackathon_site/registration/forms.py

```diff
@@ -149,6 +152,18 @@ def clean(self):
             )
         return cleaned_data
 
+    def clean_birthday(self):
+        latest_birthday = (
+            settings.EVENT_START_DATE - timedelta(days=settings.MINIMUM_AGE * 365)
```

Comment:
Ah joy, there's going to be an edge case with leap years. 

I think we're fine to install [dateutil](https://dateutil.readthedocs.io/en/stable/) and use `relativedelta`, maybe it will come in handy elsewhere as well. Other alternatives are to use 365.24 to get a bit more accurate, or construct a date object manually from parts which is ugly.

When you install it locally (`pip install python-dateutil)`, do `pip freeze | grep dateutil` and add that line to our requirements.txt where it should be alphabetically.